Rudder-Facebook/Classes/RudderFacebookIntegration.m

```diff
@@ -115,29 +131,120 @@ - (void)flush {
     [RSLogger logDebug:@"Facebook App Events Factory doesn't support Flush Call"];
 }
 
-
 #pragma mark - Utils
 
-- (NSNumber *)extractRevenue:(NSDictionary *)dictionary withKey:(NSString *)revenueKey
-{
-    id revenueProperty = nil;
-    
-    for (NSString *key in dictionary.allKeys) {
-        if ([key caseInsensitiveCompare:revenueKey] == NSOrderedSame) {
-            revenueProperty = dictionary[key];
-            break;
+- (NSString *)getFacebookEvent:(NSString *)event {
+    if ([event isEqualToString:@"Products Searched"]) {
```

Comment:
Can you use the constants of RSECommerceEvents?

Reply:
Implemented.